colors.less

```diff
@@ -0,0 +1,115 @@
+// White
```

Comment:
Was this file included by mistake?

Reply:
No this file is used since I have applied css colors to button

Reply:
Looks like the file has been removed from the commit and the new colors added to the existing file. Thanks.